src/styles/styles.js

```diff
@@ -1016,24 +1016,7 @@ const styles = {
         height: 12,
         width: 12,
         zIndex: 10,
-    },
-
-    statusIndicatorLarge: {
-        borderColor: themeColors.componentBG,
-        backgroundColor: themeColors.danger,
-        borderRadius: 8,
-        borderWidth: 2,
-        position: 'absolute',
-        right: 4,
-        bottom: 4,
-        height: 16,
-        width: 16,
-        zIndex: 10,
-    },
-
-    statusIndicatorOnline: {
-        backgroundColor: themeColors.success,
-    },
+    }),
```

Comment:
Deleting this cuz it turns out this is not being used by AvatarWithIndicator (or anywhere)